vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -948,6 +948,10 @@ public User getOrCreateUser(String email, String institution, String groupName)
 
         User user;
         try {
+            if (!email.contains("@")) {
```

Comment:
There may be another part in the code where we verify the email string is completely valid and could be reused.